packages/engine-twig-php/lib/engine_twig_php.js

```diff
@@ -90,7 +90,8 @@ const engine_twig_php = {
           if (results.ok) {
             resolve(results.html + details);
           } else {
-            reject(results.message);
+            console.log(results.message);
+            process.exit(1);
```

Comment:
```suggestion
            // make Twig rendering errors more noticeable + exit when not in dev mode (or running the `patternlab serve` command)
            if (
              process.argv.slice(1).includes('serve') ||
              process.env.NODE_ENV === 'development'
            ) {
              reject(chalk.red(results.message));
            } else {
              console.log(chalk.red(results.message));
              process.exit(1);
            }
```

Reply:
LMK what you think -- if you're good with this then I'm good with this 👍

Reply:
`patternlab serve`
![image](https://user-images.githubusercontent.com/1617209/79984074-a788f700-8476-11ea-93f2-290e4f5b470a.png)

vs

`patternlab build`
![image](https://user-images.githubusercontent.com/1617209/79984119-b66fa980-8476-11ea-8e86-57f9d045ac83.png)

Reply:
this looks like a stellar UX to me

Reply:
@EvanLovely i'll let you drive the acceptance of this

Reply:
I'm down 👍 - just committed to this branch. I would suggest similar behavior if any template engine called `reject()` from it's `renderPattern()` function. However I think this works great for now. Thanks!

Reply:
👍🏼